README.md

```diff
@@ -64,6 +64,9 @@ REACT_APP_AUTH0_AUD
 
 See the provided `.env.schema` file for more details.
 
+You may need to contact the project maintainer to get the appropriate value for
```

Comment:
We can replace with with 
> You will need to setup an application in Auth0 and use it to set the `REACT_APP_AUTH0_CLIENT_ID` variable. Make sure to configure the application to allow redirects at the local server's address (typically http://localhost:3000/callback)

Reply:
Ok so is it expected that each developer will need to setup an application in Auth0 to get their client ID? Or is this blurb for the general developer outside Kids First? In other words, are all Kids First developers expected to use the same client id?

Reply:
Devs outside Kids First should need to set up their own Auth0 app. Devs in Kids First could use the same application client id or set them up individually.